htmresearch/algorithms/column_pooler.py

```diff
@@ -69,6 +72,22 @@ def __init__(self,
     @param  sdrSize (int)
             The number of active cells in an object SDR
 
+    @param  onlineLearning (Bool)
+            Whether or not the column pooler should learn in online mode.
+
+    @param  learningTolerance (float)
```

Comment:
I like this from a simplicity standpoint, but it will be important to make sure that maxSdrSize and minSdrSize default to sdrSize, no matter what value for that parameter was passed in.  Otherwise, a user could easily break everything by passing in sdrSize = 20 and not passing a minSdrSize, which would default to 40.  I think it's easy to deal with this by making their defaults None, and just setting them to sdrSize if None during initialization.